Makefile

```diff
@@ -16,7 +16,7 @@ build:
 .PHONY: install
 install: build
 	install -D -t $(DESTDIR)$(PREFIX)/bin bin/{ore,kola,plume}
-	install -D -m 0755 -t $(DESTDIR)$(PREFIX)/lib/kola/$(GOARCH) bin/$(GOARCH)/kolet
+	install -D -m 0755 -t $(DESTDIR)$(PREFIX)/lib/kola/$(GOARCH) bin/kolet
```

Comment:
Oh ugh, sorry - it looks like this worked for me because my build tree was dirty with the binary in the old place.